pykern/pkcollections.py

```diff
@@ -99,9 +101,65 @@ def pkdel(self, name, default=None):
             except KeyError:
                 pass
 
+    def pkmerge(self, to_merge):
+        """Add `to_merge` to `self`
+
+        `self` will have references to `to_merge` after this call.
```

Comment:
@e-carlin @mkeilman What do you think about destructive copies? I just ran into this being a problem when coding rsconf. Perhaps it could be a flag so that fconf can use it for its purposes.

Reply:
I added make_copy as a parameter in 3f60b12. It makes no sense to make it destructive by default.

Reply:
I think it's a good idea to give the option; I'm not certain what the default behavior should be though.

Reply:
Too slow :)